packages/v4/src/content/design-guidelines/components/clipboard-copy/clipboard-copy.md

```diff
@@ -38,6 +37,15 @@ An expandable clipboard copy allows for long lines of text to be stored in an ex
 
 <img src="./img/expandable.png" alt="Example of an expandable clipboard copy component" width="455"/>
 
+**Inline**
+
+An inline clipboard copy is placed directly inline with other text or used on it's own when you want a lighter weight clipboard copy treatment.
```

Comment:
Should be its not it's!

Reply:
Thanks for catching that, I'll update it.